lib/API/DX/Device.cpp

```diff
@@ -753,7 +1134,7 @@ class DXDevice : public offloadtest::Device {
       // Zero any remaining bytes
       if (R.size() < CBVSize) {
         void *ExtraData = static_cast<char *>(ResDataPtr) + R.size();
-        memset(ExtraData, 0, CBVSize - R.size() - 1);
+        memset(ExtraData, 0, CBVSize - R.size());
```

Comment:
Is this an unrelated bug fix? I'm nervous about a change like this without accompanying tests.

Reply:
Are you still intending to undo this change, or did you find it is actually required?